src/MCPClient/lib/clientScripts/sanitize_names.py

```diff
@@ -21,100 +21,82 @@
 # @subpackage MCPClient
 # @author Joseph Perry <joseph@artefactual.com>
 
-import string
 import os
-from shutil import move as rename
-import sys
-import unicodedata
+import re
+import shutil
+
+from scandir import scandir
 from unidecode import unidecode
-from archivematicaFunctions import unicodeToStr
+
+from archivematicaFunctions import strToUnicode
 
 VERSION = "1.10." + "$Id$".split(" ")[1]
-valid = "-_.()" + string.ascii_letters + string.digits
-replacementChar = "_"
 
+# Letters, digits and a few punctuation characters
+ALLOWED_CHARS = re.compile(r"[^a-zA-Z0-9\-_.\(\)]")
```

Comment:
As part of archivematica/issues#315 it looks like we need to update the third paragraph [here](https://github.com/artefactual/archivematica-docs/blob/1.9/user-manual/transfer/clean-up-names.rst). The regular expression translates to the same set of characters as I read it so it's likely just the line we link to.

Reply:
PR: https://github.com/artefactual/archivematica-docs/pull/292